README.md

```diff
@@ -50,27 +57,59 @@ Generated `config` directory when also using `--cega` option:
 config
 ├── cega.config
 ├── cega.json
+├── certs
+│   ├── cega-mq.ca.crt
+│   ├── cega-mq.ca.key
+│   ├── cega-users.ca.crt
+│   ├── cega-users.ca.key
+│   ├── dataedge.ca.crt
+│   ├── dataedge.ca.key
+│   ├── db.ca.crt
+│   ├── db.ca.key
+│   ├── filedatabase.ca.crt
+│   ├── filedatabase.ca.key
+│   ├── finalize.ca.crt
+│   ├── finalize.ca.key
+│   ├── htsget.ca.crt
+│   ├── htsget.ca.key
+│   ├── inbox.ca.crt
+│   ├── inbox.ca.key
+│   ├── ingest.ca.crt
+│   ├── ingest.ca.key
+│   ├── keys.ca.crt
+│   ├── keys.ca.key
+│   ├── mq.ca.crt
```

Comment:
MQ can act as both server and client so we should be specific that these are the server cert and key by setting the name to `mq-server.ca.crt`

Reply:
Done.